src/pods/test-sentences/test-sentences.mappers.ts

```diff
@@ -0,0 +1,21 @@
+import * as sentenceApi from 'core/sentences';
+import * as sentenceVm from './test.sentences.vm';
+import * as verbApi from 'core/verbs/global-verbs.api';
+import { splitSentence } from './test-sentences.business';
+
+export const mapFromSentenceApiToSentenceVm = (
+  sentenceApi: sentenceApi.SentenceEntityApi,
+  verbApi: verbApi.VerbEntityApi
```

Comment:
This should not be called _verbApi_ if I read this I think you are talking about an api to make calls, in this case is just an entity, call it _verbEntityApi_

Reply:
done